src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -549,24 +558,28 @@ const WorkflowView = _.flow(
             ]),
             span({ style: { color: colors.darkBlue[0], fontSize: 24 } }, name)
           ]),
+          isRedacted && div({ style: { color: colors.orange[0], fontSize: 16, fontWeight: 500, marginTop: '0.5rem' } }, [
+            'The selected snapshot of the referenced tool has been redacted. You will not be able to run an analysis until you select another snapshot.'
+          ]),
           div({ style: { marginTop: '0.5rem' } }, [
             'Snapshot ',
             sourceRepo === 'agora' ?
-              div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 60  } }, [
+              div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 60 } }, [
                 h(Select, {
                   isDisabled: !!Utils.editWorkspaceError(ws),
                   isClearable: false,
                   isSearchable: false,
                   value: methodVersion,
                   getOptionLabel: ({ value }) => Utils.normalizeLabel(value),
-                  options: _.isEmpty(snapshotIds) ? [methodVersion] : snapshotIds,
+                  options: _.uniq([...snapshotIds, methodVersion]).sort(),
```

Comment:
So there's a weird behavior if you go from redacted snapshot to non-redacted where the redacted snapshot disappears from the drop down menu. Changing the snapshot appears to be uni-directional from redacted snapshot-->non-redacted snapshot. It's not clear why the redacted snapshot disappears. 

In FC, the behavior is similar, but you can still see that there's a redacted snapshot in the drop down, just not one you can click on to switch back to.